origin.spec

```diff
@@ -0,0 +1,420 @@
+%{_bindir}/oc
+%{_bindir}/openshift
+%{_bindir}/openshift-router
+%{_bindir}/openshift-deploy
+%{_bindir}/openshift-sti-build
+%{_bindir}/openshift-docker-build
+%{_bindir}/origin
+%{_bindir}/atomic-enterprise
+%{_bindir}/oadm
+%{_bindir}/kubectl
+%{_bindir}/kubernetes
+%{_bindir}/kubelet
+%{_bindir}/kube-proxy
+%{_bindir}/kube-apiserver
+%{_bindir}/kube-controller-manager
+%{_bindir}/kube-scheduler
+%{_sharedstatedir}/origin
+%{_sysconfdir}/bash_completion.d/*
+%dir %config(noreplace) %{_sysconfdir}/origin
+
+%pre
+# If /etc/openshift exists symlink it to /etc/origin
+if [ -d "%{_sysconfdir}/openshift" ]; then
+  ln -s %{_sysconfdir}/openshift %{_sysconfdir}/origin
+fi
+
+%files master
+%defattr(-,root,root,-)
+%{_unitdir}/%{name}-master.service
+%config(noreplace) %{_sysconfdir}/sysconfig/%{name}-master
+%config(noreplace) %{_sysconfdir}/origin/master
+%ghost %config(noreplace) %{_sysconfdir}/origin/admin.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/admin.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/admin.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/ca.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/ca.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/ca.serial.txt
+%ghost %config(noreplace) %{_sysconfdir}/origin/etcd.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/etcd.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master-config.yaml
+%ghost %config(noreplace) %{_sysconfdir}/origin/master.etcd-client.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master.etcd-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master.kubelet-client.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master.kubelet-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-master.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-master.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-master.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-registry.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-registry.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-registry.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-router.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-router.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/openshift-router.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/policy.json
+%ghost %config(noreplace) %{_sysconfdir}/origin/serviceaccounts.private.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/serviceaccounts.public.key
+
+%post master
+%systemd_post %{basename:%{name}-master.service}
+# Create master configs if they do not exist
+if [ ! -e %{_sysconfdir}/origin/master/master-config.yaml ]; then
+  %{_bindir}/openshift start master --write-config=%{_sysconfdir}/origin/master
```

Comment:
Ug. Right. So at the moment the code and the packaging are not aligned. The code requires ```atomic-enterprise``` to be the name of the binary that runs if Atomic Enterprise is to be in use. Otherwise it should be called ```openshift```. I guess it could be done via symlink and systemd unit.